### PR TITLE
translating /docs/tasks/tools/install-minikube/ to Indonesia

### PR DESCRIPTION
**Pull request by dedyks**
Creating install-minikube.md to translate
/docs/tasks/tools/install-minikube/ into Bahasa Indonesia

### CODE REVIEW

File: content/id/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+### Instalasi sebuah Hypervisor
+
+Apabila kamu belum memiliki sebuah Hypervisor terinstal, Instalasi salah satu sekarang:
+
+• [Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_install)
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+{{< note >}}
+Hyper-V hanya dapat berjalan diatas tiga versi Windws 10: Windows 10 Enterprise, Windows 10 Professional, dan Windows 10 Education.
```
Comment (sftim) [inline content/id/docs/tasks/tools/install-minikube.md:174]: nit:
```suggestion
Hyper-V hanya dapat berjalan diatas tiga versi Windows 10: Windows 10 Enterprise, Windows 10 Professional, dan Windows 10 Education.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+{{< tabs name="minikube_before_you_begin" >}}
+{{% tab name="Linux" %}}
+Untuk mengecek apabila virtualisasi didukung di Linux, jalankan perintah berikut dan pastikan keluarannya tidak kosong:
+```
+grep -E --color 'vmx|svm' /proc/cpuinfo
+```
+{{% /tab %}}
+
+{{% tab name="macOS" %}}
+Untuk mengecek apabila virtualisasi didukung di Linux, jalankan perintah berikut di terminal kamu.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:27]:
```suggestion
Untuk mengecek apabila virtualisasi didukung di macOS, jalankan perintah berikut di terminal kamu.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+```
+sysctl -a | grep -E --color 'machdep.cpu.features|VMX'
+```
+Apabila kamu melihat `VMX` pada hasil keluaran (berwarna), artinya fitur VT-x sudah diaktifkan di mesin kamu.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:31]:
```suggestion
Apabila kamu melihat `VMX` pada hasil keluaran (seharusnya berwarna), artinya fitur VT-x sudah diaktifkan di mesin kamu.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+{{% capture steps %}}
+
+# Menginstal minikube
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="Linux" %}}
+
+### Instalasi kubectl
+
+Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi di [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux).
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:67]:
```suggestion
Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi pada halaman [Instalasi dan Konfigurasi kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux).
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+
+### Instalasi sebuah Hypervisor
+
+Apabila kamu belum menginstal sebuah Hypervisor, Instalasi salah satu sekarang:
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:71]:
```suggestion
Apabila kamu belum menginstal sebuah Hypervisor, silakan instal salah satu:
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+
+• [KVM](https://www.linux-kvm.org/), yang menggunakan QEMU
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+Minikube juga mendukung sebuah opsi `--driver=none` untuk menjalankan komponen - komponen Kubernetes diatas *host* dan tidak didalam VM. Untuk menggunakan *driver* ini memerlukan [Docker](https://www.docker.com/products/docker-desktop) dan sebuah lingkungan Linux bukan sebuah hypervisor.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:77]:
```suggestion
Minikube juga mendukung sebuah opsi `--driver=none` untuk menjalankan komponen-komponen Kubernetes pada *host* dan tidak didalam VM. Untuk menggunakan *driver* ini maka diperrlukan [Docker](https://www.docker.com/products/docker-desktop) dan sebuah lingkungan Linux yang bukan sebuah hypervisor.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+
+Apabila kamu menggunakan `none` *driver* di Debian atau turunannya, gunakan  paket `.deb` untuk Docker daripada menggunakan paket *snap*, karena paket *snap* tidak berfungsi dengan Minikube.
+Kamu bisa mengunduh paket `.deb` dari [Docker](https://www.docker.com/products/docker-desktop).
+
+{{< caution >}}
+*Driver* VM `none` dapat menyebabkan masalah pada keamanan dan kehilangan data. Sebelum menggunakan *driver* VM, periksa [dokumentasi ini](https://minikube.sigs.k8s.io/docs/reference/drivers/none/) untuk informasi lebih lanjut.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:83]:
```suggestion
*Driver* VM `none` dapat menyebabkan masalah pada keamanan dan kehilangan data. Sebelum menggunakan opsi `--driver=none`, periksa [dokumentasi ini](https://minikube.sigs.k8s.io/docs/reference/drivers/none/) untuk informasi lebih lanjut.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+{{< /caution >}}
+
+Minikube juga mendukung opsi `vm-driver=podman` yang mirip dengan *driver* Docker. Podman yang berjalan dengan hak istimewa *superuser* adalah cara terbaik untuk memastikan kontainer - kontainer kamu memiliki akses penuh ke semua fitur yang ada di sistem kamu..
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:86]:
```suggestion
Minikube juga mendukung opsi `vm-driver=podman` yang mirip dengan *driver* Docker. Podman yang berjalan dengan hak istimewa *superuser* adalah cara terbaik untuk memastikan kontainer-kontainer kamu memiliki akses penuh ke semua fitur yang ada di sistem kamu..
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+
+{{< caution >}}
+*Driver* `podman` memerlukan *root* untuk menjalankan kontainer karena akun pengguna reguler tidak memiliki akses penuh ke semua fitur sistem operasi yang mungkin kontainer perlukan.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:89]:
```suggestion
*Driver* `podman` mengharuskan kontainer untuk berjalan dengan akses *root* karena akun pengguna reguler tidak memiliki akses penuh ke semua fitur sistem operasi yang mungkin diperlukan oleh kontainer.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+{{< /caution >}}
+
+### Instalasi Minikube menggunakan paket
+
+Tersedia paket *experimental* untuk Minikube, kamu bisa menemukan paket untuk Linux (AMD64) dari halaman  [*releases*](https://github.com/kubernetes/minikube/releases) Minikube di GitHub.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:94]:
```suggestion
Tersedia paket uji coba untuk Minikube, kamu bisa menemukan paket untuk Linux (AMD64) dari halaman  [*releases*](https://github.com/kubernetes/minikube/releases) Minikube di GitHub.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 11]
```diff
@@ -0,0 +1,257 @@
...
+
+Jika kamu tidak menginstal melalui sebuah paket, kamu bisa mengunduh sebuah *stand-alone binary* dan menggunakannya.
+
+
+```shell
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 \
+  && chmod +x minikube
+```
+
+Berikut adalah cara mudah untuk menambahkan *executable* Minikube ke *path* kamu.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:108]:
```suggestion
Berikut adalah cara mudah untuk menambahkan program Minikube ke *path* kamu.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 12]
```diff
@@ -0,0 +1,257 @@
...
+
+```shell
+brew install minikube
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+### Instalasi kubectl
+
+Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi di [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos).
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:127]:
```suggestion
Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi pada halaman [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos).
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 13]
```diff
@@ -0,0 +1,257 @@
...
+
+### Instalasi sebuah Hypervisor
+
+Apabila kamu belum memiliki sebuah Hypervisor terinstal, Instalasi salah satu sekarang:
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:131]:
```suggestion
Apabila kamu belum menginstal sebuah Hypervisor, silakan instal salah satu:
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 14]
```diff
@@ -0,0 +1,257 @@
...
+
+{{% /tab %}}
+{{% tab name="Windows" %}}
+### Instalasi kubectl
+
+Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi di [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
+
+### Instalasi sebuah Hypervisor
+
+Apabila kamu belum memiliki sebuah Hypervisor terinstal, Instalasi salah satu sekarang:
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:167]:
```suggestion
Apabila kamu belum menginstal sebuah Hypervisor, silakan instal salah satu:
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 15]
```diff
@@ -0,0 +1,257 @@
...
+
+```shell
+sudo mv minikube /usr/local/bin
+```
+
+{{% /tab %}}
+{{% tab name="Windows" %}}
+### Instalasi kubectl
+
+Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi di [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:163]:
```suggestion
Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi pada halaman [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 16]
```diff
@@ -0,0 +1,257 @@
...
+```
+
+Kamu juga bisa menginstalnya dengan mengunduh *stand-alone binary*:
+
+```shell
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64 \
+  && chmod +x minikube
+```
+
+Berikut adalah cara mudah untuk menambahkan *executable* Minikube ke *path* kamu.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:153]:
```suggestion
Berikut adalah cara mudah untuk menambahkan program Minikube ke *path* kamu.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 17]
```diff
@@ -0,0 +1,257 @@
...
+
+### Instalasi Minikube menggunakan Chocolatey
+
+Cara paling mudah untuk menginstal Minikube di Windows adalah menggunakan [Chocolatey](https://chocolatey.org/) (jalankan sebagai administrator):
+
+```shell
+choco install minikube
+```
+
+Setelah Minikube telah selesai diinstal, tutup sesi *CLI* dan matikan ulang. Minikube akan ditambahkan ke *path* kamu secara otomatis.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:185]:
```suggestion
Setelah Minikube telah selesai diinstal, tutup sesi *CLI* dan hidupkan ulang. Minikube akan ditambahkan ke *path* kamu secara otomatis.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 18]
```diff
@@ -0,0 +1,257 @@
...
+
+### Instalasi Minikube menggunakan sebuah *installer executable*
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:187]:
```suggestion
### Instalasi Minikube menggunakan sebuah program penginstal 
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 19]
```diff
@@ -0,0 +1,257 @@
...
+
+Untuk menginstal Minikube secara manual di Windows menggunakan [Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal), unduh [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe) dan jalankan *installer*.
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:189]:
```suggestion
Untuk menginstal Minikube secara manual di Windows menggunakan [Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal), unduh [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe) dan jalankan program penginstal.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 20]
```diff
@@ -0,0 +1,257 @@
...
+* [Menjalanakan Kubernetes secara lokal dengan Minikube](/docs/setup/learning-environment/minikube/)
+
+{{% /capture %}}
+
+## Memastikan instalasi
+
+Untuk memastikan keberhasilan kedua instalasi hypervisor dan Minikube, kamu bisa menjalankan perintah berikut untuk memulai klaster Kubernetes lokal:
+{{< note >}}
+
+Untuk pengaturan  `--driver` dengan `minikube start`, masukkan nama hypervisor `<driver_name>` yang kamu instal dengan huruf kecil seperti yang ditunjukan dibawah. Daftar lengkap nilai `--driver` tersedia di [dokumentasi menentukan *driver* VM](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:212]:
```suggestion
Untuk pengaturan  `--driver` dengan `minikube start`, masukkan nama hypervisor `<driver_name>` yang kamu instal dengan huruf kecil seperti yang ditunjukan dibawah. Daftar lengkap nilai `--driver` tersedia di [dokumentasi menentukan *driver* VM](/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 21]
```diff
@@ -0,0 +1,257 @@
...
+minikube start
+```
+
+dan `minikube start` memberikan *error*:
+```
+machine does not exist
+```
+
+maka kamu dapat membersihkan lokal *state* minikube:
+then you need to clear minikube's local state:
```
Comment (wahyuoi) [inline content/id/docs/tasks/tools/install-minikube.md:254]:
```suggestion
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 22]
```diff
@@ -0,0 +1,257 @@
...
+Berikut adalah cara mudah untuk menambahkan *executable* Minikube ke *path* kamu.
+
+```shell
+sudo mkdir -p /usr/local/bin/
+sudo install minikube /usr/local/bin/
+```
+
+### Instalasi Minikube menggunakan Homebrew
+
+Sebagai alternatif lain, kamu bisa menginstal Minikube menggunakan Linux [Homebrew](https://docs.brew.sh/Homebrew-on-Linux):
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:117]:
```suggestion
Sebagai alternatif, kamu bisa menginstal Minikube menggunakan Linux [Homebrew](https://docs.brew.sh/Homebrew-on-Linux):
```

"Alternatif" already implies "lain". We can omit the redundancy

File: content/id/docs/tasks/tools/install-minikube.md (hunk 12 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:127]:
```suggestion
Pastikan kamu mempunyai kubectl yang terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi pada laman [Menginstal dan Menyiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-macos).
```

Same as the previous one about kubectl.

File: content/id/docs/tasks/tools/install-minikube.md (hunk 13 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:131]:
```suggestion
Apabila kamu belum menginstal sebuah Hypervisor, silakan instal salah satu  dari:
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 23]
```diff
@@ -0,0 +1,257 @@
...
+Apabila kamu belum memiliki sebuah Hypervisor terinstal, Instalasi salah satu sekarang:
+
+• [HyperKit](https://github.com/moby/hyperkit)
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+• [VMware Fusion](https://www.vmware.com/products/fusion)
+
+### Instalasi Minikube
+Cara paling mudah untuk menginstal Minikube di macOS adalah menggunakan [Homebrew](https://brew.sh):
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:140]:
```suggestion
Cara paling mudah untuk menginstal Minikube pada macOS adalah menggunakan [Homebrew](https://brew.sh):
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 24]
```diff
@@ -0,0 +1,257 @@
...
+
+```shell
+brew install minikube
+```
+
+Kamu juga bisa menginstalnya dengan mengunduh *stand-alone binary*:
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:146]:
```suggestion
Kamu juga bisa menginstalnya dengan mengunduh _stand-alone binary_-nya:
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 16 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:153]:
```suggestion
Berikut adalah cara mudah untuk menambahkan program Minikube ke _path_ kamu.
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 15 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:163]:
```suggestion
Pastikan kamu mempunyai kubectl yang terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
```

Same as previous comment about kubectl.

File: content/id/docs/tasks/tools/install-minikube.md [hunk 25]
```diff
@@ -0,0 +1,257 @@
...
+sudo mv minikube /usr/local/bin
+```
+
+{{% /tab %}}
+{{% tab name="Windows" %}}
+### Instalasi kubectl
+
+Pastikan kamu mempunyai kubectl terinstal. Kamu bisa menginstal kubectl berdasarkan instruksi di [Instalasi dan Mempersiapkan kubectl](/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
+
+### Instalasi sebuah Hypervisor
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:165]:
```suggestion
### Menginstal sebuah Hypervisor
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 14 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:167]:
```suggestion
Apabila kamu belum menginstal sebuah Hypervisor, silakan instal salah satu dari:
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 26]
```diff
@@ -0,0 +1,257 @@
...
+
+Apabila kamu belum memiliki sebuah Hypervisor terinstal, Instalasi salah satu sekarang:
+
+• [Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_install)
+
+• [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+{{< note >}}
+Hyper-V hanya dapat berjalan diatas tiga versi Windows 10: Windows 10 Enterprise, Windows 10 Professional, dan Windows 10 Education.
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:174]:
```suggestion
Hyper-V hanya dapat berjalan pada tiga versi dari Windows 10: Windows 10 Enterprise, Windows 10 Professional, dan Windows 10 Education.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 27]
```diff
@@ -0,0 +1,257 @@
...
+{{< /note >}}
+
+### Instalasi Minikube menggunakan Chocolatey
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:177]:
```suggestion
### Menginstal Minikube menggunakan Chocolatey
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 28]
```diff
@@ -0,0 +1,257 @@
...
+
+Cara paling mudah untuk menginstal Minikube di Windows adalah menggunakan [Chocolatey](https://chocolatey.org/) (jalankan sebagai administrator):
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:179]:
```suggestion
Cara paling mudah untuk menginstal Minikube pada Windows adalah menggunakan [Chocolatey](https://chocolatey.org/) (jalankan sebagai administrator):
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 17 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:185]:
```suggestion
Setelah Minikube telah selesai diinstal, tutup sesi CLI dan hidupkan ulang CLI-nya. Minikube akan ditambahkan ke _path_ kamu secara otomatis.
```

Akronim tidak perlu dimiringkan. CLI.

File: content/id/docs/tasks/tools/install-minikube.md (hunk 18 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:187]:
```suggestion
### Menginstal Minikube menggunakan sebuah program penginstal 
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 19 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:189]:
```suggestion
Untuk menginstal Minikube secara manual pada Windows menggunakan [Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal), unduh [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe) dan jalankan program penginstal tersebut.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 29]
```diff
@@ -0,0 +1,257 @@
...
+choco install minikube
+```
+
+Setelah Minikube telah selesai diinstal, tutup sesi *CLI* dan matikan ulang. Minikube akan ditambahkan ke *path* kamu secara otomatis.
+
+### Instalasi Minikube menggunakan sebuah *installer executable*
+
+Untuk menginstal Minikube secara manual di Windows menggunakan [Windows Installer](https://docs.microsoft.com/en-us/windows/desktop/msi/windows-installer-portal), unduh [`minikube-installer.exe`](https://github.com/kubernetes/minikube/releases/latest/download/minikube-installer.exe) dan jalankan *installer*.
+
+### Installasi Minikube melalui unduh langsung
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:191]:
```suggestion
### Menginstal Minikube melalui pengunduhan langsung
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 30]
```diff
@@ -0,0 +1,257 @@
...
+
+Untuk menginstal Minikube secara manual di Windows, unduh [`minikube-windows-amd64`](https://github.com/kubernetes/minikube/releases/latest), ubah nama menjadi `minikube.exe`, dan tambahkan ke *path* kamu.
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:193]:
```suggestion
Untuk menginstal Minikube secara manual pada Windows, unduh [`minikube-windows-amd64`](https://github.com/kubernetes/minikube/releases/latest), ubah nama menjadi `minikube.exe`, dan tambahkan ke _path_ kamu.
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 31]
```diff
@@ -0,0 +1,257 @@
...
+
+{{% capture whatsnext %}}
+
+* [Menjalanakan Kubernetes secara lokal dengan Minikube](/docs/setup/learning-environment/minikube/)
+
+{{% /capture %}}
+
+## Memastikan instalasi
+
+Untuk memastikan keberhasilan kedua instalasi hypervisor dan Minikube, kamu bisa menjalankan perintah berikut untuk memulai klaster Kubernetes lokal:
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:209]:
```suggestion
Untuk memastikan keberhasilan kedua instalasi hypervisor dan Minikube, kamu bisa menjalankan perintah berikut untuk memulai sebuah klaster Kubernetes lokal:
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 20 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:212]:
```suggestion
Untuk pengaturan  `--driver` dengan `minikube start`, masukkan nama hypervisor `<driver_name>` yang kamu instal dengan huruf kecil seperti yang ditunjukkan di bawah. Daftar lengkap nilai `--driver` tersedia di [dokumentasi menentukan _driver_ VM](/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
```

Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:212]: Penggunaan "di" sebagai preposisi tempat harus dipisah. Mis. "di bawah", "di sana", dll.
Imbuhan "di-" untuk membentuk verba pasif digabung. Mis. "dibawa", "dicuci", dll.

Ref:
https://twitter.com/ivanlanin/status/903979453763641344

File: content/id/docs/tasks/tools/install-minikube.md [hunk 32]
```diff
@@ -0,0 +1,257 @@
...
+
+Untuk pengaturan  `--driver` dengan `minikube start`, masukkan nama hypervisor `<driver_name>` yang kamu instal dengan huruf kecil seperti yang ditunjukan dibawah. Daftar lengkap nilai `--driver` tersedia di [dokumentasi menentukan *driver* VM](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
+
+{{< /note >}}
+
+```shell
+minikube start --driver=<driver_name>
+```
+
+Setalah `minikube start` selesai, jalankan perintah dibawah untuk mengecek status klaster:
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:220]:
```suggestion
Setalah `minikube start` selesai, jalankan perintah di bawah untuk mengecek status klaster:
```

File: content/id/docs/tasks/tools/install-minikube.md [hunk 33]
```diff
@@ -0,0 +1,257 @@
...
+
+```shell
+minikube status
+```
+
+Jika klaster berjalan, keluaran dari `minikube status` akan mirip seperti ini:
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:226]:
```suggestion
Jika klasternya berjalan, keluaran dari `minikube status` akan mirip seperti ini:
```

Or.. "klaster kamu"

File: content/id/docs/tasks/tools/install-minikube.md [hunk 34]
```diff
@@ -0,0 +1,257 @@
...
+
+```
+host: Running
+kubelet: Running
+apiserver: Running
+kubeconfig: Configured
+```
+
+Setelah kamu memastikan bahwa Minikube berjalan sesuai dengan hypervisor yang telah kamu pilih, kamu dapat melanjutkan untuk memakai Minikube atau menghentikan klaster kamu. Untuk menghentikan klaster, jalankan:
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:235]:
```suggestion
Setelah kamu memastikan bahwa Minikube berjalan sesuai dengan hypervisor yang telah kamu pilih, kamu dapat melanjutkan untuk menggunakan Minikube atau menghentikan klaster kamu. Untuk menghentikan klaster, jalankan:
```

"Pakai" adalah kata yang dipakai pada bahasa cakapan. Untuk bahasa tertulis, "guna" lebih tepat.

Lihat "cak" pada kategori katanya:
https://kbbi.kemdikbud.go.id/entri/pakai

File: content/id/docs/tasks/tools/install-minikube.md [hunk 35]
```diff
@@ -0,0 +1,257 @@
...
+```
+
+## Membersihkan *state* lokal {#cleanup-local-state}
+
+Jika sebelumnya kamu pernah menginstal Minikube, dan menjalankan:
+```shell
+minikube start
+```
+
+dan `minikube start` memberikan *error*:
```
Comment (phanama) [inline content/id/docs/tasks/tools/install-minikube.md:248]:
```suggestion
dan `minikube start` memberikan pesan kesalahan:
```

File: content/id/docs/tasks/tools/install-minikube.md (hunk 20 above)
Comment (phanama) [reply content/id/docs/tasks/tools/install-minikube.md:212]: Ditujukan -> ditunjukkan

**Comment by girikuncoro** [comment]
@dedyks thanks for your contribution, all seems to be good to me and you have addressed all feedbacks, can you remove the Draft mark of this PR and squash the commit into 1 with good message commit?

all good but will hold the merge

/lgtm
/hold

**Comment by girikuncoro** [comment]
@dedyks thanks for great work! hope you learning something by localizing this page, feel free to pick up other pages which interest you :)

/unhold
/lgtm
/approve

**Pull request merged**